openshift-install-powervs

```diff
@@ -831,6 +832,19 @@ function variables_nodes {
     no_of_bastion=1
   fi
 
+  if [ ! -z $flavor ]; then
```

Comment:
Use `-n` instead of `! -z`.

Reply:
Also use "$flavor" or [[ -n $flavor ]] or the will fail if flavor is empty.
Always have your code [shellchecked](http://shellcheck.net)